lib/cast/cast_proxy.js

```diff
@@ -188,12 +188,10 @@ shaka.cast.CastProxy = class extends shaka.util.FakeEventTarget {
    * @export
    */
   async cast() {
-    const initState = this.getInitState_();
-
     // TODO: transfer manually-selected tracks?
     // TODO: transfer side-loaded text tracks?
 
-    await this.sender_.cast(initState);
+    await this.sender_.cast();
```

Comment:
This completely changes the start time of the video, among other things.  I don't think this is the right change.

Reply:
For example, if you are 15 minutes into an episode of a show, then decide to cast to your TV to finish it, you don't want it to start over at 0.

Reply:
The time is kept, see the changes of lib/cast/cast_sender.js